openssl/ecdh.go

```diff
@@ -69,144 +84,128 @@ func NewPrivateKeyECDH(curve string, bytes []byte) (*PrivateKeyECDH, error) {
 	if err != nil {
 		return nil, err
 	}
+	b := bytesToBN(bytes)
+	if b == nil {
+		return nil, newOpenSSLError("BN_bin2bn failed")
```

Comment:
Generally the pattern is the error message matches the name of the func that failed. In this case it doesn't. Wanted to double check that is intentional here.

Reply:
`bytesToBN` internally calls `BN_bin2bn`, and the pattern is to put the openssl function name. I'll remove the ` failed` prefix, which is not what we usually write.